lessons/pydata/pandas_groupby/index.ipynb

```diff
@@ -0,0 +1,3297 @@
+{
```

Comment:
<p>Imho nevědí, co je <code>next</code>  ani <code>iter</code>.  Takže zase odbočka?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/73/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<p>No proto ta odbočka :-) Nevim, jak jinak vysvětlit tu iteraci.</p><p> </p><p>jedině že bych udělal</p><p> </p><p>for element in groups:</p><p>  print(element)</p><p>  break</p>

Reply:
Ale já bych je klidně nechal, jen se musejí vysvětlit :-)